Source/Assets/PlayFabEditorExtensions/Editor/Scripts/PlayFabEditorSDK/PlayFabEditorHttp.cs

```diff
@@ -52,11 +52,11 @@ private static void WriteResultFile(string url, Action<string> resultCallback, b
 
             if (api.Contains("/Server/") || api.Contains("/Admin/"))
             {
-                if (PlayFabEditorDataService.ActiveTitle == null)
-                    throw new Exception("Title not found, please select your title in the PlayFab EdEx Panel, Setting Tabs");
-
-                if (string.IsNullOrEmpty(PlayFabEditorDataService.ActiveTitle.SecretKey))
-                    throw new Exception("Title Secret key not found, please clear and re-select your title in the PlayFab EdEx Panel, Setting Tabs");
+                if (PlayFabEditorDataService.ActiveTitle == null || string.IsNullOrEmpty(PlayFabEditorDataService.ActiveTitle.SecretKey))
+                {
+                    PlayFabEditorDataService.RefreshStudiosList();
+                    return;
```

Comment:
Is it meant to just silently return now without throwing an exception?

Reply:
Yes. The panel is going to be doing this quite frequently now, so I can't spam the warning.
In effect, to resolve the problem I'm fixing, I have to give up reporting this particular problem to the user.
I don't really think it costs the user anything...